entrypoint.py

```diff
@@ -284,24 +285,37 @@ async def prpc_wshandler(request):
     payload = cmd["packet"].get("payload", "")
 
     scheduler = cmd.get("scheduler", "IMMEDIATE")
+    queue_length = int(cmd.get("queue_length", 100))  # No unlimited queues.
+
+    if queue_length <= 0:
+        await ws.close(message=b"queue_length must be > 0.")
+        return
 
     tm = a0.TopicManager(container="api", prpc_client_aliases={"topic": cmd})
 
     ns = types.SimpleNamespace()
     ns.loop = asyncio.get_event_loop()
-    ns.q = asyncio.Queue()
+    ns.cond = asyncio.Condition()
+    ns.q = collections.deque(maxlen=queue_length)
```

Comment:
Use asyncio.Queue as before, but replace
```py
ns.q = asyncio.Queue()
...
ns.loop.call_soon_threadsafe(ns.q.put_nowait, (pkt, done))
```
with
```py
ns.q = asyncio.Queue(maxsize=queue_length)
...
ns.loop.run_coroutine_threadsafe(ns.q.put, (pkt, done))
```

https://docs.python.org/3/library/asyncio-queue.html

No need for the condition variable

Reply:
Was discussed, intention is to silently drop oldest messages for now.